modules/govuk_jenkins/templates/jobs/transition_load_site_config.yaml.erb

```diff
@@ -22,15 +27,13 @@
           artifact-num-to-keep: 5
     builders:
         - shell: |
-            <%- if scope.lookupvar('::aws_migration') %>
             HOSTNAME=$(govuk_node_list -c backend --single-node)
-            <%- else -%>
-            HOSTNAME="backend-1.backend"
-            <%- end -%>
             rsync --delete -avz -e ssh ./ deploy@$HOSTNAME:/var/apps/transition/data/transition-config
             ssh deploy@$HOSTNAME "cd /var/apps/transition && govuk_setenv transition bundle exec rake import:all:orgs_sites_hosts"
```

Comment:
I think this is the task which updates the redirects used by bouncer, as it's what actually loads the data from the site configuration yaml file

Reply:
I see what's going on

Reply:
Yup 👍 - previously the other task did this as well.